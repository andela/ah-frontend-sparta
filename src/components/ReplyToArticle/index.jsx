import React from 'react';
import '../ReplyToComment';

const ReplyToArticle = ({ onCommentChanged, postComment }) => (
  <div className="form-group shadow-textarea" style={{ overflow: 'hidden' }}>
    <textarea
      className="form-control commentReplyTextarea z-depth-1"
      rows="5"
      name="comment"
      onChange={onCommentChanged}
      placeholder="Comment on Article"
    />
    <button type="submit" className="commentReplyButtn" onClick={postComment}>
      POST
      {' '}
    </button>
  </div>
);

export default ReplyToArticle;
