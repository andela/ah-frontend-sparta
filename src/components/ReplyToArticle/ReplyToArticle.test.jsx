import React from 'react';
import renderer from 'react-test-renderer';
import ReplyToArticle from './index';

test('ReplyToArticle test', () => {
  const wrapper = renderer.create(<ReplyToArticle />);
  expect(wrapper).toMatchSnapshot();
});
