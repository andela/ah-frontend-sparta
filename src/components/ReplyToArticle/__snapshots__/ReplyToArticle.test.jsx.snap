// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReplyToArticle test 1`] = `
<div
  className="form-group shadow-textarea"
  style={
    Object {
      "overflow": "hidden",
    }
  }
>
  <textarea
    className="form-control commentReplyTextarea z-depth-1"
    name="comment"
    placeholder="Comment on Article"
    rows="5"
  />
  <button
    className="commentReplyButtn"
    type="submit"
  >
    POST
     
  </button>
</div>
`;
